packages/aws-lambda/test/integration_test/test_definition.js

```diff
@@ -1086,7 +1088,9 @@ function registerTests(handlerDefinitionPath, reduced) {
         // * the heartbeat request succeed, and the
         // * data being sent via the extension (instead of being sent to the back end directly).
         const spansFromExtension = await control.getSpansFromExtension();
-        expect(spansFromExtension).to.have.length(2);
+        // This test fails for the v24 prerelease due to a deprecation warning for `url.parse()`,
```

Comment:
Please find it here: https://cloud.ibm.com/devops/pipelines/tekton/c2cd6a8d-ea5a-47b0-913e-cd172d63833f/runs/b6f2187b-5991-4544-94dd-fb716521b4f6/test-ci-aws-lambda-task/run-test-group?env_id=ibm:yp:eu-de

Reply:
Thanks

>@instana/aws-lambda: (node:4727) [DEP0169] DeprecationWarning: `url.parse()` behavior is not standardized and prone to errors that have security implications. Use the WHATWG URL API instead. CVEs are not issued for `url.parse()` vulnerabilities.
@instana/aws-lambda: (Use `node --trace-deprecation ...` to show where the warning was created)

But the warning is everywhere and other tests are also not failing 🤔

Reply:
https://github.com/instana/nodejs/pull/1430#pullrequestreview-2420572655

Reply:
Hmm, Interesting. 

This warning is traced as a span in all cases but we don't have span length assertions in many tests. Thats why they are all passing.
The case I fixed had a span length check and it was failing. This is a temporary fix before we do [this](https://jsw.ibm.com/browse/INSTA-18579).

Reply:
Thanks for pointing out the span lengths checks. I have added this information to the ticket https://jsw.ibm.com/browse/INSTA-18579

I still would like to revert this quick fix because we don't know yet why there is suddenly less spans and the change is too hard to maintain.